std_msgs/Header header
shape_msgs/Mesh mesh
float64[]       u
float64[]       v
